config/cesm/machines/config_batch.xml

```diff
@@ -556,6 +556,7 @@
     <job name="case.st_archive">
       <template>template.st_archive</template>
       <task_count>1</task_count>
+      <walltime>0:10:00</walltime>
```

Comment:
Is 10 minutes long enough for all possible case.st_archive runs?

Reply:
I'd probably err on the side of making it longer - maybe 20 or 30 min?